homeassistant/components/auth/__init__.py

```diff
@@ -68,13 +68,16 @@
 from homeassistant.components.http.ban import log_invalid_auth
 from homeassistant.components.http.data_validator import RequestDataValidator
 from homeassistant.components.http.view import HomeAssistantView
-from homeassistant.core import callback
+from homeassistant.core import callback, HomeAssistant
 from homeassistant.util import dt as dt_util
+
 from . import indieauth
 from . import login_flow
+from . import mfa_setup_flow
+from . import util
 
 DOMAIN = 'auth'
-DEPENDENCIES = ['http']
+DEPENDENCIES = ['http', 'websocket_api']
```

Comment:
Websocket API commands can be registered without loading the websocket API. It is not a dependency.

If the user wants the websocket API, the commands will be available.